Docker fails: `block in materialize': Could not find minitest-5.14.2 in any of the sources (Bundler::GemNotFound)

I get the following error on both Mac and Linux right now:

```
$ make refresh
....
$ make
rm -rf /tmp/jekyll-docker
mkdir -p /tmp/jekyll-docker
cp Gemfile /tmp/jekyll-docker
docker build /tmp/jekyll-docker -f Dockerfile -t jekyll-image
[+] Building 1.5s (9/9) FINISHED
 => [internal] load build definition from Dockerfile                                                                                                                                                       0.0s
 => => transferring dockerfile: 37B                                                                                                                                                                        0.0s
 => [internal] load .dockerignore                                                                                                                                                                          0.0s
 => => transferring context: 2B                                                                                                                                                                            0.0s
 => [internal] load metadata for docker.io/library/ubuntu:20.04                                                                                                                                            1.4s
 => [1/4] FROM docker.io/library/ubuntu:20.04@sha256:c95a8e48bf88e9849f3e0f723d9f49fa12c5a00cfc6e60d2bc99d87555295e4c                                                                                      0.0s
 => [internal] load build context                                                                                                                                                                          0.0s
 => => transferring context: 107B                                                                                                                                                                          0.0s
 => CACHED [2/4] RUN apt-get -y update     && apt-get install -y         build-essential         git         ruby         ruby-dev         zlib1g-dev                                                      0.0s
 => CACHED [3/4] ADD Gemfile .                                                                                                                                                                             0.0s
 => CACHED [4/4] RUN gem install bundler     && gem update --system     && bundle install                                                                                                                  0.0s
 => exporting to image                                                                                                                                                                                     0.0s
 => => exporting layers                                                                                                                                                                                    0.0s
 => => writing image sha256:80c36d2215a768bf478764c90d5e173b274b82b379e802acd8aa996c7ffbf90f                                                                                                               0.0s
 => => naming to docker.io/library/jekyll-image                                                                                                                                                            0.0s
docker run --rm -p 4000:4000 -v /Users/agude/Project/MiniFate:/workspace -w /workspace jekyll-image bundle exec jekyll serve
/var/lib/gems/2.7.0/gems/bundler-2.2.4/lib/bundler/spec_set.rb:87:in `block in materialize': Could not find minitest-5.14.2 in any of the sources (Bundler::GemNotFound)
	from /var/lib/gems/2.7.0/gems/bundler-2.2.4/lib/bundler/spec_set.rb:81:in `map!'
	from /var/lib/gems/2.7.0/gems/bundler-2.2.4/lib/bundler/spec_set.rb:81:in `materialize'
	from /var/lib/gems/2.7.0/gems/bundler-2.2.4/lib/bundler/definition.rb:175:in `specs'
	from /var/lib/gems/2.7.0/gems/bundler-2.2.4/lib/bundler/definition.rb:245:in `specs_for'
	from /var/lib/gems/2.7.0/gems/bundler-2.2.4/lib/bundler/definition.rb:227:in `requested_specs'
	from /var/lib/gems/2.7.0/gems/bundler-2.2.4/lib/bundler/runtime.rb:91:in `block in definition_method'
	from /var/lib/gems/2.7.0/gems/bundler-2.2.4/lib/bundler/runtime.rb:20:in `setup'
	from /var/lib/gems/2.7.0/gems/bundler-2.2.4/lib/bundler.rb:149:in `setup'
	from /var/lib/gems/2.7.0/gems/bundler-2.2.4/lib/bundler/setup.rb:20:in `block in <top (required)>'
	from /var/lib/gems/2.7.0/gems/bundler-2.2.4/lib/bundler/ui/shell.rb:136:in `with_level'
	from /var/lib/gems/2.7.0/gems/bundler-2.2.4/lib/bundler/ui/shell.rb:88:in `silence'
	from /var/lib/gems/2.7.0/gems/bundler-2.2.4/lib/bundler/setup.rb:20:in `<top (required)>'
	from /usr/lib/ruby/2.7.0/rubygems/core_ext/kernel_require.rb:92:in `require'
	from /usr/lib/ruby/2.7.0/rubygems/core_ext/kernel_require.rb:92:in `require'
make: *** [serve] Error 1
exit: 2
```

--- a/Dockerfile
+++ b/Dockerfile
@@ -22,6 +22,7 @@ RUN apt-get -y update \
 ADD Gemfile .
 RUN gem install bundler \
     && gem update --system \
-    && bundle install
+    && bundle install \
+    && bundle update  # This prevents Bug #39, would love to remove it
 
 CMD ["/bin/bash"]

